mirascope/core/mistral/_utils/_load_api_key.py

```diff
@@ -0,0 +1,6 @@
+import os
+
+
+def load_api_key() -> str:
+    """Load the API key from the standard environment variable."""
+    return os.environ.get("MISTRAL_API_KEY", "")
```

Comment:
nit: I think we should actually just remove this method entirely and just use os directly, no reason to wrap this